PennMobile/Setup + Navigation/RootViewController.swift

```diff
@@ -133,6 +128,25 @@ class RootViewController: UIViewController, NotificationRequestable {
             
         }
         
+        //Request to fetch Two Factor Code again if the app failed to fetch it last time
+        if shouldFetchTwoFactorCode() {
+            DispatchQueue.main.async {
+                let alert = UIAlertController(title: "Two-Step Code Not Fetched", message: "We were unable to fetch your Two-Step code last time. Do you want to try again?", preferredStyle: .alert)
+                alert.addAction(UIAlertAction(title: "No", style: .default, handler: { _ in
+                    alert.dismiss(animated: true, completion: nil)
+                    UserDefaults.standard.setTwoFactorEnabled(to: false);
+                }))
+                
+                alert.addAction(UIAlertAction(title: "Yes", style: .default, handler: { _ in
+                    let twc = TwoFactorWebviewController()
+                    UserDefaults.standard.setTwoFactorEnabledAgain(to: true)
```

Comment:
I'm wary of us trying to manage so many states because it will make it hard to debug in the future. Rather than having another user defaults variable for "enabled again," why don't we just set "twoFactorEnabled" to false?